generators/skill/templates/function/python2/src/__main__.py

```diff
@@ -6,11 +6,15 @@ def main(req):
     try:
         result = main_impl(req['payload'])
     except KeyError as e:
-        result = { 'error': 'Missing required key {0}'.format(e.args[0]) }
-    return { 'payload': result }
+        result = {'error': 'Missing required key {0}'.format(e.args[0])}
```

Comment:
Minor thing, but instead of using `e.args[0]` why not just use `e`? Exceptions have `__str__()` and `__repr__()` defined.